.travis.yml

```diff
@@ -14,13 +16,8 @@ env:
     - TWIG_EXT=yes
 
 before_script:
-    - if [ "$TWIG_EXT" == "yes" ]; then sh -c "cd ext/twig && phpize && ./configure --enable-twig && make && sudo make install"; fi
+    - if [ "$TWIG_EXT" == "yes" ]; then sh -c "cd ext/twig && phpize && ./configure --enable-twig && make && make install"; fi
```

Comment:
this should be done in 1.x IMO, to benefit from the faster infrastructure there too